lib/pure/os.nim

```diff
@@ -1592,10 +1592,17 @@ proc getFilePermissions*(filename: string): set[FilePermission] {.
     else:
       result = {fpUserExec..fpOthersRead}
 
-proc setFilePermissions*(filename: string, permissions: set[FilePermission]) {.
-  rtl, extern: "nos$1", tags: [WriteDirEffect], noWeirdTarget.} =
+proc setFilePermissions*(filename: string, permissions: set[FilePermission],
+                         followSymlinks = true)
+  {.rtl, extern: "nos$1", tags: [ReadDirEffect, WriteDirEffect],
+   noWeirdTarget.} =
   ## Sets the file permissions for `filename`.
   ##
+  ## If ``followSymlinks`` set to true (default) and ``filename`` points to a
```

Comment:
https://nim-lang.github.io/Nim/contributing.html
> In nim sources, prefer single backticks to double backticks since it's simpler and nim doc supports it (even in rst files with nim rst2html).